app/models/comment.rb

```diff
@@ -1,2 +1,10 @@
 class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  validates :text, presence: true
+
+  def update_posts_counter
+    post.increment!(:comments_counter)
+  end
```

Comment:
- [ ] [OPTIONAL] Kindly change the method name to `#update_comments_counter` instead of `#update_posts_counter`.
- [ ] [OPTIONAL] You can use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - `after_save` to call the method every time a new `comment` has been saved to the database, then make the method private.